Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.cpp

```diff
@@ -171,6 +173,20 @@ void InjectedBundle::resetOriginAccessAllowLists()
     WebProcess::singleton().ensureNetworkProcessConnection().connection().send(Messages::NetworkConnectionToWebProcess::ResetOriginAccessAllowLists { }, 0);
 }
 
+void InjectedBundle::addMixedContentWhitelistEntry(const String& origin, const String& domain)
```

Comment:
why we need to call it via bundle?

Reply:
That's what they do for the CORS whitelist, so it felt like the "right way" instead of calling it directly from WebKitWebExtension. Lets see what upstream says